api/types/swarm/container.go

```diff
@@ -116,4 +116,5 @@ type ContainerSpec struct {
 	CapabilityAdd  []string            `json:",omitempty"`
 	CapabilityDrop []string            `json:",omitempty"`
 	Ulimits        []*units.Ulimit     `json:",omitempty"`
+	OomScoreAdj    int64               `json:",omitempty"`
```

Comment:
I'm torn. On the one hand, coding style dictates that this field should be spelled `OOMScoreAdj` on account of ["OOM" being an initialism](https://go.dev/wiki/CodeReviewComments#initialisms) for "out of memory." On the other hand, we have existing precedent _in the API_ for this spelling, with `(container.HostConfig).OomScoreAdj`.
https://github.com/moby/moby/blob/ff5cc18482bea99a9baba27112474eff156df778/api/types/container/hostconfig.go#L443
I am okay with either spelling. Anyone have a strong opinion one way or the other?

Reply:
Match the existing API field name. No reason to make Swarmkit any *more* different than it already is.

Reply:
Yeah, I hate the Oom, but agree if it's matching the engine API, it's fine.